app/controllers/hub/outgoing_emails_controller.rb

```diff
@@ -14,10 +14,43 @@ def create
       redirect_to hub_client_messages_path(client_id: @client, anchor: "last-item")
     end
 
+    def unsubscribe_email
```

Comment:
Sorry I know we did do that hub route work around but looking at it more I do think it makes more sense to put it in a controller thats not the hub and make it more like the state-file `NotificationsSettingsController`